pyne/ace.pyx

```diff
@@ -1555,10 +1735,121 @@ class Reaction(object):
         self.TY = None     # Neutron release
         self.IE = 0        # Energy grid index
         self.sigma = []    # Cross section values
-
-    def broaden(self, T_high):
-        pass
-
+        self.sqrt_pi_inv = 1. / np.sqrt(np.pi)
```

Comment:
Don't see why you'd want sqrt_pi_inv to be an instance attribute. I'd just calculate this on the fly or at most, make it a private module variable. Also, I'm pretty sure plain `math.sqrt(math.pi)` is going to be faster than using `np.sqrt` since there is overhead for checking the type of the argument.